script/e2e-test.sh

```diff
@@ -91,12 +103,17 @@ installOLM() {
 
     kubectl apply -f ${url}/crds.yaml
 
-    # The Pod that populates the catalog gets OOM Killed due to very low limits
-    # This has been fixed here: https://github.com/operator-framework/operator-lifecycle-manager/pull/1389
-    # But the fix has not been published yet. To workaround the issue we are using a newer image
-    # This will be fixed in a version > 0.14.2
-    kubectl apply -f "${ROOT_DIR}/script/manifests/olm.yaml"
-
+    if [[ "${GKE_BRANCH-}" != "1.15" ]]; then
```

Comment:
Should this code still be here if we're not installing OLM for 1.15 either?

Reply:
right, I forgot to delete it